examples/build.gradle

```diff
@@ -1,18 +1,19 @@
-apply plugin: 'com.android.application'
+apply plugin: 'com.android.library'
 
-android {
-    compileSdkVersion 24
-    buildToolsVersion '26.0.2'
+repositories {
+    mavenCentral()
+    jcenter()
```

Comment:
Do we still need jcenter here? Since you also removed it in the other Gradle files.

Reply:
Yes, because tus android is still on Jcenter